javascript-mjs/draft.md

```diff
@@ -2,7 +2,6 @@
 title: ECMAScript Media Types Updates
 abbrev:
 docname: draft-ietf-dispatch-javascript-mjs-02
-date: 2017
```

Comment:
Removing this `date:` field means `kramdown-rfc2629` inserts the current date into the XML and the eventual TXT/HTML/PDF et al.